packages/mobile/src/index.tsx

```diff
@@ -5,12 +5,16 @@ import 'fast-text-encoding';
 import { Navigation } from '~/navigation/navigation';
 import { store } from '~/store/store';
 import { NavigationDarkTheme, NavigationTheme } from '~/config/config';
-import { useColorScheme } from '~/hooks/hooks';
+import { useColorScheme, useEffect } from '~/hooks/hooks';
+import RNBootSplash from 'react-native-bootsplash';
 import { i18 } from './localization/localization';
 
 i18();
 
 const App: FC = () => {
+  useEffect(() => {
+    RNBootSplash.hide({ fade: true });
+  }, []);
```

Comment:
Again, this belongs to the `NavigatorContainer`'s `onReady` callback

Reply:
fixed